src/docs/admin/references/NodeSourceInfrastructureReference.adoc

```diff
@@ -138,82 +138,112 @@ Image). For the Resource Manager to use EC2 instances as computing
 
-+
-Using this configuration, you can start a Resource Manager and a
-Scheduler using the */bin/proactive-server* script. An EC2 NodeSource can
-now be added using the *Create Node Source* panel in the Resource Manager or the command line interface:
+-   *rmHostname*: The hostname or the public IP address of the Resource Manager.
+        This address needs to be accessible from the AWS cloud.
+        For example, `try.activeeon.com`.
+
+-   *connectorIaasURL*: Connector-iaas is a ProActive service used to interact with IaaS like AWS.
+        By default it runs on the following URL _rmHostname/connector-iaas_.
+        For example, `\http://try.activeeon.com:8080/connector-iaas`.
 
-    $ PROACTIVE_HOME/bin/proactive-client --createns ec2 -infrastructure org.ow2.proactive.resourcemanager.nodesource.infrastructure.AWSEC2Infrastructure aws_key aws_secret_key rmDomain connectorIaasURL image numberOfInstances numberOfNodesPerInstance downloadCommand additionalProperties minRam minCores
+-   *nodeJarURL*: The full URL path of the _node.jar_ to download the ProActive node.jar on each new created AWS instance.
+        The URL needs to be accessible from the AWS cloud.
+        For example, `try.activeeon.com/rest/node.jar`.
 
-As AWS is a paying service, when the Scheduler is stopped normally (without removing the created infrastructure), all the created AWS instances will be terminated. And when the Scheduler is restarted, these instances will be re-configured as per the previous settings.
+-   *additionalProperties*: Additional Java command properties to be added when starting each ProActive node JVM in AWS instances (e.g. `-Dpropertyna
+me=propertyvalue`). This is an optional property.
+
+-   *nodeTimeout*: The estimated startup time of the nodes (expressed in millisecond). After this timeout expired, the node is considered as lost.
+
+Using this configuration, you can start a ProActive Resource Manager and Scheduler using the */bin/proactive-server* script.
+An AWS EC2 NodeSource can now be added using the *Create Node Source* panel in the Resource Manager Portal or the command line interface:
+
+    $ PROACTIVE_HOME/bin/proactive-client --createns ec2 -infrastructure org.ow2.proactive.resourcemanager.nodesource.infrastructure.AWSEC2Infrastructure awsKey awsSecretKey numberOfInstances numberOfNodesPerInstance image vmUsername vmKeyPairName vmPrivateKey ram cores securityGroupIds subnetId rmHostname connectorIaasURL nodeJarURL additionalProperties nodeTimeout
+
+As AWS is a paying service, when the ProActive server is stopped normally (without removing the created infrastructure), all the created AWS instances will be terminated. And when the ProActive server is restarted, these instances will be re-configured as per the previous settings.
```

Comment:
`And when the ...` A sentence should not start with `And`, remove it.

Reply:
Done.